community/modules/scheduler/htcondor-configure/variables.tf

```diff
@@ -24,6 +24,22 @@ variable "deployment_name" {
   type        = string
 }
 
+variable "labels" {
+  description = "Labels to add to resources. List key, value pairs."
+  type        = map(string)
+}
+
+variable "region" {
+  description = "Default region for creating resources"
+  type        = string
+}
+
+variable "subnetwork_self_link" {
+  description = "The self link of the subnetwork in which Central Managers will be placed."
+  type        = string
+  default     = null
```

Comment:
In retrospect I think this needs to have no default value. Please review rest of PR but I will address.